plugins/scaffolder/src/components/MultistepJsonForm/MultistepJsonForm.tsx

```diff
@@ -123,6 +126,8 @@ export const MultistepJsonForm = (props: Props) => {
     finishButtonLabel,
     layouts,
   } = props;
+  const { templateName } = useRouteRefParams(selectedTemplateRouteRef);
```

Comment:
I see it's just used for a fallback label text so it's not a big deal, but just wanted to note here that it doesn't include the namespace.